### DISCUSSION TITLE
Using jsartoolkit ES6 as a npm module for multi markers ?

### DISCUSSION THREAD

**Discussion by acronie** [webarkit/jsartoolkitNFT#396] [Q&A]
Hello,

I'm using jsartoolkit in various projects since 3 years and I managed to setup a nice dev chain with an example for multi-markers tracking I found somewhere (`artoolkit.wasm_multi_worker.js`, adding `artoolkit_wasm.js` and `artoolkit_wasm.wasm`). It works great (really!!), but I'm still using a kind of "old school" structure with an `importmap` in my index.html. I wanted to rewrite my app to be more "modern", using vite.js and ES6 modules (including three.js and others tools I need), but when I looked at the last update of jsartoolkit I couldn't find an example to help me setting the project up and running. I'm fighting with it for several days now and I could make markers loading work by using `artoolkitNFT_multi_ES6.worker.js` and some part of my previous "main.js" script, but I can't get markers recognition to be processed... rather frustrating!

Does anyone have a basic sample with npm ES6 module multi tracking with workers ?

Thanks

**Comment by kalwalt** [comment]
Hi @acronie sorry for the delay, I think you are referring to webarkit/jsartoolkit5 (?) not to this repository for sure, you are also referring to loading in an ES6 module that jsartoolkit5 has not. Maybe you could use instead [artoolkit5-js (ar-js-org branch)](https://github.com/AR-js-org/artoolkit5-js/tree/ar-js-org), it depends if you want use pattern markers, barcode markers or NFT?
